### ISSUE TITLE
open link in new page

### ISSUE THREAD

**Issue by dpolivaev** [github/cmark-gfm#205]
how can add target="_blank" to markdown links [test](http://test.com) in github?!

See https://github.com/mojombo/github-flavored-markdown/issues/28 which seems to be the wrong project and check the discussion there.

Is this project the right one?

**Comment by wesinator** [comment]
Yes, I think it would be good to change the autolink behaviour to open links in new tab by default:
`<a href="http://example.com" target="_blank" rel="noopener"` 

In my experience, good to open links in new tab preserving the page that I'm on.

Note that _target blank now implies `noopener` in newer browser versions (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target), but noopener should still be added explicitly for older browser versions that aren't on the latest W3C spec.

Cheers,
